src/aws/interfaces/vault-definition-model.interface.ts

```diff
@@ -0,0 +1,23 @@
+import { BouncerType, Network, Protocol, VaultBehavior, VaultState, VaultVersion } from '@badger-dao/sdk';
+
+import { Stage } from '../../config/enums/stage.enum';
+
+export interface IVaultDefinition {
+  address: string;
+  behavior: VaultBehavior;
+  bouncer: BouncerType;
+  chain: Network;
+  client: string;
+  createdAt: number;
+  depositToken: string;
+  isNew: boolean;
+  isMigrating: boolean;
+  isProduction: number;
+  name: string;
+  protocol: Protocol;
+  releasedAt: number;
+  stage: Stage;
```

Comment:
whats the difference between stage and isProduction

Reply:
mb u can come up with better naming, on the time being, `stage` of the vault depends on `VaultState`, which can be 
```ts
export declare enum VaultState {
    Open = "open",
    Guarded = "guarded",
    Experimental = "experimental",
    Discontinued = "discontinued",
    Featured = "featured"
}
```
and the expression is `vault.state === VaultState.Experimental ? Stage.Staging : Stage.Production`.

But `isProduction` field is more about, are we still getting this vault from the onchain call
`const prdVaults = await this.registry.getProductionVaults();`

So from the `sdk` perspective, every vault that is in onchain response from `registry` is `Production`.

Reply:
VaultState is for state not stage right?